script/Deploy.s.sol

```diff
@@ -128,23 +127,21 @@ contract MaciVotingScript is Script {
             treeDepths: maciEnvVariables.treeDepths,
             messageBatchSize: maciEnvVariables.messageBatchSize
         });
-
         return (params, tokenSettings, mintSettings);
```

Comment:
`return (params, tokenSettings, mintSettings);`

we can remove this line because we are already declaring the return values on the top of the function (my bad 😂 I think I left this one before)